projects/1.18/assets/ob-aquamirae/aquamirae/lang/zh_cn.json

```
@@ -0,0 +1,232 @@
+{
+  "ability.abyssal_armor_full_1": "全套装能力 §8搭配头盔>遭受致命攻击后，死亡推迟#1秒§7。>效果生效期间，你造成的伤害减少80%。",
+  "ability.abyssal_armor_full_2": "全套装能力 §8搭配头冠>造成周围敌对生物失明，并降低它们>50%的移动速度。",
+  "ability.abyssal_armor_half": "半套装加成>§9+25% 护甲值>§9+4 盔甲韧性",
+  "ability.coral_lance": "压制>对海船墓地生物造成#1%§7更多伤害。",
+  "ability.cost.none": "",
+  "ability.dagger_of_greed_1": "盗贼>被攻击的村民与掠夺者掉落绿宝石。",
+  "ability.dagger_of_greed_2": "惩戒>攻击村民时有几率获得凶兆。",
+  "ability.dagger_of_greed_3": "§d特殊锻造>§5由远古铁匠锻造，且不可修复。",
+  "ability.divider": "切断>降低目标#1秒§7内5%§7的>生命值上限，累计最多10次。",
+  "ability.fin_cutter": "复仇>根据你已损失的生命值，造成>#1%§7更多伤害（最高#2%§7）。",
+  "ability.maze_rose": "主动技能>生成#2秒§7的环刃，造成>#1§7近战伤害。",
+  "ability.poisoned_blade": "感染>给予被攻击的敌人#1秒§7的>中毒效果。",
+  "ability.poisoned_chakra": "主动技能>生成#2秒§7的环刃，造成>#1§7伤害并给予中毒效果。",
+  "ability.remnants_saber": "适应>在水中时造成#1%§7额外伤害。",
+  "ability.terrible_armor_full": "全套装能力>给予攻击你的敌人#1秒§7的>中毒效果。",
+  "ability.terrible_armor_half": "半套装能力>在水中遭受攻击将提升你>#2秒§7内#1%§7的游泳速度。",
+  "ability.terrible_sword": "嗜血之眼>如果攻击未杀死敌方，>你将受到#1§7点伤害。",
+  "ability.three_bolt_armor_full": "全套装能力>在冰雪迷阵水域使你免>受冰冻。",
+  "ability.three_bolt_armor_half": "半套装能力>只对上衣和头盔生效：>上衣在水下为你提供氧气。",
+  "ability.whisper_of_the_abyss": "穿透>降低目标#1秒§7内护甲值10%。>累计最多5次。",
+  "advancements.aquamirae.abyssal_armor.descr": "制作并装备一整套（耀眼）深海装备",
+  "advancements.aquamirae.abyssal_armor.title": "深海装备",
+  "advancements.aquamirae.catch_golden_moth.descr": "用瓶子抓住一只闪亮的飞蛾",
+  "advancements.aquamirae.catch_golden_moth.title": "亮闪闪！",
+  "advancements.aquamirae.consume_poseidons_breakfast.descr": "烹饪并食用波塞冬的早餐",
+  "advancements.aquamirae.consume_poseidons_breakfast.title": "海洋的力量",
+  "advancements.aquamirae.consume_sea_stew.descr": "烹饪并食用海之炖菜",
+  "advancements.aquamirae.consume_sea_stew.title": "海的味道我知道",
+  "advancements.aquamirae.find_ice_maze.descr": "在冻洋生物群系找到冰雪迷阵。",
+  "advancements.aquamirae.find_ice_maze.title": "冰雪迷阵",
+  "advancements.aquamirae.obtain_anglers_fang.descr": "获得一个鮟鱇尖牙",
+  "advancements.aquamirae.obtain_anglers_fang.title": "完美实例",
+  "advancements.aquamirae.obtain_dagger_of_greed.descr": "找到贪婪之匕首",
+  "advancements.aquamirae.obtain_dagger_of_greed.title": "贪婪",
+  "advancements.aquamirae.obtain_esca.descr": "获得“小灯笼”",
+  "advancements.aquamirae.obtain_esca.title": "黑暗中的光",
+  "advancements.aquamirae.obtain_fin.descr": "获得一个鳍（Fin）",
+  "advancements.aquamirae.obtain_fin.title": "好似一片扇叶（Fan）",
```

原文标题：`Almost Like a Fan`
原文描述：`Obtain a Fin`

我个人对双关/谐音梗的处理方式是：尽量符合原文；如果不能符合原文，就在符合文本需要表达的内容的情况下，做成本地化双关/谐音梗
此处我的处理方式建议如下：
```suggestion
  "advancements.aquamirae.obtain_fin.descr": "获得一个鳍",
  "advancements.aquamirae.obtain_fin.title": "听着像旗一样",
```
```suggestion
  "advancements.aquamirae.obtain_fin.descr": "获得一个鳍",
  "advancements.aquamirae.obtain_fin.title": "整整“鳍鳍”",
```